Automattic/wp-calypso
# Pricing page rework: Fix styles on WPCOM Plans page

#### Proposed Changes

* Fix styles for Plans page on WPCOM
* Fix Hero and Data for monthly plans

#### Testing Instructions

- Create a JN site and connect Jetpack
- Do any one of these
    * Click on Calypso live link below and wait for the redirect to complete, then goto `/plans/:site?flags=jetpack/pricing-page-rework-v1`
    * or boot up this PR 
        * Run `git fetch && git checkout add/jp-pricing-rework/fix-wpcom-styles`
        * Run `yarn start`
        * Goto `http://calypso.localhost:3000/plans/:site?flags=jetpack/pricing-page-rework-v1`
- Confirm that you see the new pricing UI
- Confirm that the design matches Figma designs
- Navigate to other Calypso blue pages
- Confirm that there is no regression like changed button style/color etc.

#### Pre-merge Checklist






Fixes 1202796695664022-as-1202898236955402/f



#### When a site has a plan
| BEFORE | AFTER |
|--------|-------|
|<img width="1602" alt="Screenshot 2022-09-02 at 5 30 12 PM" src="https://user-images.githubusercontent.com/18226415/188140961-890f2834-b296-4270-8a3b-8531a9eee698.png">|<img width="1616" alt="Screenshot 2022-09-02 at 5 26 56 PM" src="https://user-images.githubusercontent.com/18226415/188140947-5109539b-a14e-4c63-a5d7-effe4f33ccd6.png">|
|<img width="1601" alt="Screenshot 2022-09-02 at 5 30 35 PM" src="https://user-images.githubusercontent.com/18226415/188140967-95680c55-9700-48de-b2c2-d9404209b152.png">|<img width="1602" alt="Screenshot 2022-09-02 at 5 31 20 PM" src="https://user-images.githubusercontent.com/18226415/188140976-33fbc472-9a0c-4c9c-aab2-031ca9c643eb.png">|

#### When a site does not have a plan
| BEFORE | AFTER |
|--------|-------|
|<img width="1463" alt="Screenshot 2022-09-02 at 5 35 03 PM" src="https://user-images.githubusercontent.com/18226415/188140984-902e95ab-46e9-44db-92f8-8d3cd4795c67.png">|<img width="1464" alt="Screenshot 2022-09-02 at 5 35 49 PM" src="https://user-images.githubusercontent.com/18226415/188140990-2394c3d7-fd6d-4752-9f3a-517bbf0a95d3.png">|


#### Mobile view
| BEFORE | AFTER |
|--------|-------|
|<img width="366" alt="Screenshot 2022-09-02 at 5 58 00 PM" src="https://user-images.githubusercontent.com/18226415/188143035-a7d62748-6a96-42da-87e3-26e7c46f4072.png">|<img width="366" alt="Screenshot 2022-09-02 at 5 59 08 PM" src="https://user-images.githubusercontent.com/18226415/188143041-f2534123-95ff-4441-9796-b38de64d6607.png">|


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202898236955402
